dataset-snippets/snippets_setup.py

```diff
@@ -0,0 +1,24 @@
+!pip3 install --upgrade --user pandas_profiling statsmodels
+
+import os
+import numpy as np
+import pandas as pd
+import pandas_profiling
+import plotnine
+from plotnine import *  # Provides a ggplot-like interface to matplotlib.
+
+## Plot setup.
+theme_set(theme_bw(base_size = 11)) # Default theme for plots.
+
+def get_boxplot_fun_data(df):
+  """Returns a data frame with a y position and a label, for use annotating ggplot boxplots.
+
+  Args:
+    d: A data frame.
+  Returns:
+    A data frame with column y as max and column label as length.
+  """
+  d = {'y': max(df), 'label': f'N = {len(df)}'}
+  return(pd.DataFrame(data=d, index=[0]))
+
+# NOTE: if you get any errors from this cell, restart your kernel and run it again.
```

Comment:
Is there anything reproducible here that we should file?

Reply:
There is a package install and a package upgrade in the pip command at the top.

It will always cause the snippet to fail at first because a Python kernel has to be restarted to make use of the new/updated packages.